FWCore/Concurrency/test/streamGrapher_stallMonitor_cfg.py

```diff
@@ -23,4 +23,4 @@
                                       numberOfStreams = cms.untracked.uint32(4),
                                       numberOfThreads = cms.untracked.uint32(5))
 
-process.add_(cms.Service("Tracer", printTimestamps = cms.untracked.bool(True)))
+process.add_(cms.Service("StallMonitor", fileName = cms.untracked.string("stallMonitor.log")))
```

Comment:
My understanding was that the `StallMonitor` was replacing the `Tracer` for looking for stalls, hence the change.  However, if you'd like to retain both for comparison reasons, I can add the `Tracer` service back.